docs/src/app/components/pages/components/date-picker.jsx

```diff
@@ -5,6 +5,11 @@ const Code = require('date-picker-code');
 const CodeExample = require('../../code-example/code-example');
 const CodeBlock = require('../../code-example/code-block');
 
+if (!window.Intl) {
+  require('intl');
```

Comment:
Oh, sorry, now I get it :sweat_smile: I'll probably make a PR generally improving the contributing experience, so I'll fix that too.